src/translations/de.json

```diff
@@ -28,13 +49,15 @@
     "pause": "Pause",
     "stop": "Stop",
     "return_to_base": "Dock",
-    "locate": "Staubsauger lokalisieren"
+    "locate": "Staubsauger lokalisieren",
+    "not_available": "Staubsaugerroboter ist nicht verfügbar"
   },
   "error": {
+    "invalid_config": "Ungeültige Konfiguration",
     "missing_entity": "Angabe der Entität ist erforderlich!"
   },
   "warning": {
-    "actions_array": ""
+    "actions_array": "WARNUNG: 'actions' ist dafür vorgesehen, die Standardaktionen für vorhandene Schaltflächen zu überschreiben. Wenn Sie zusätzliche Aktionen hinzufügen möchten, verwenden Sie stattdessen die Option 'Shortcuts'."
```

Comment:
`shortcuts` instead of `Shortcuts` as this is a key

Reply:
Changed - thanks